src/openforms/appointments/tests/test_admin.py

```diff
@@ -11,13 +14,36 @@
     SuperUserFactory,
     UserFactory,
 )
+from openforms.plugins.validators import PluginExistsValidator
+from openforms.utils.tests.cache import clear_caches
 
 from ..contrib.demo.plugin import DemoAppointment
+from ..fields import AppointmentBackendChoiceField
 from ..models import AppointmentsConfig
 from ..registry import Registry
 from .factories import AppointmentInfoFactory
 
 
+@contextmanager
+def patch_registry(register):
+    field = cast(
+        AppointmentBackendChoiceField, AppointmentsConfig._meta.get_field("plugin")
+    )
+    validators = [
+        validator
+        for validator in field.validators
+        if isinstance(validator, PluginExistsValidator)
+    ]
+    assert len(validators) == 1, "Expected only one PluginValidator"
+    validator = validators[0]
+    with patch("openforms.appointments.admin.register", register), patch(
+        "openforms.appointments.utils.register", register
+    ), patch.object(field, "registry", register), patch.object(
+        validator, "registry", register
+    ):
```

Comment:
```suggestion
    with (
        patch("openforms.appointments.admin.register", register),
        patch("openforms.appointments.utils.register", register),
        patch.object(field, "registry", register),
        patch.object(validator, "registry", register),
    ):

```

Sometimes you have to hold black's hand a bit; let it know we love it.

Reply:
I *knew* there was a trick, I just couldn't remember what it was, thanks!